event/models.py

```diff
@@ -8,8 +10,31 @@ def __str__(self) -> str:
         return self.name
 
 
+class EventLocation(models.Model):
+    name = models.CharField(max_length=150, blank=False, null=False)
+    # it is nullable as the location may be not always findable
+    google_maps_location = models.URLField(
```

Comment:
Add url validation for google map
eg: ^(https:\/\/)?(www\.)?(google\.com\/maps|goo\.gl\/maps)\/.*$

either in model or serializer to keep rigid contorl, avoid xss, to enhance seo

Reply:
I mean admins will put it and google does have shorter URLs as well.